test/functional/wallet_mnemonicbits.py

```diff
@@ -24,7 +24,40 @@ def run_test(self):
         self.stop_node(0)
         self.nodes[0].assert_start_raises_init_error(['-mnemonicbits=123'], "Error: Invalid '-mnemonicbits'. Allowed values: 128, 160, 192, 224, 256.")
         self.start_node(0)
-        assert_equal(len(self.nodes[0].dumphdinfo()["mnemonic"].split()), 12)  # 12 words by default
+
+        mnemonic_pre = self.nodes[0].listdescriptors(True)['descriptors'][1]["mnemonic"] if self.options.descriptors else self.nodes[0].dumphdinfo()["mnemonic"]
+
+        self.nodes[0].encryptwallet('pass')
+        self.nodes[0].walletpassphrase('pass', 100)
+        if self.options.descriptors:
+            assert "mnemonic" not in self.nodes[0].listdescriptors()['descriptors'][0]
+            assert "mnemonic" in self.nodes[0].listdescriptors(True)['descriptors'][0]
+
+            descriptors = self.nodes[0].listdescriptors(True)['descriptors']
+            assert_equal(descriptors[0]['mnemonic'], descriptors[1]['mnemonic'])
+
+            mnemonic_count = 0
+            found_in_encrypted = 0
+            for desc in descriptors:
+                if 'mnemonic' not in desc:
+                    # skip imported coinbase private key
+                    continue
+                assert_equal(len(desc['mnemonic'].split()), 12)
+                mnemonic_count += 1
+                if desc['mnemonic'] == mnemonic_pre:
+                    found_in_encrypted += 1
+                    assert desc['active']
+            # there should 5 descriptors in total
+            # one of them imported private key for coinbase without mnemonic
+            # encryption of descriptor wallet creates new private keys,
+            # it should be 2 active and 2 inactive mnemonics
```

Comment:
should adjust this text now

Reply:
I fixed text.
There's one more fix of CI failure.

Strangely, order of the descriptors on CI and on my localhost is different and it caused a failure on CI only; but on localhost it never happens (I run hundreds of times and never see it).